doc/source/changed_in_57.rst

```diff
@@ -0,0 +1,263 @@
+ * `innodb_buffer_pool_populate <https://www.percona.com/doc/percona-server/5.6/performance/innodb_numa_support.html#innodb_buffer_pool_populate>`_ server option and `numa_interleave <https://www.percona.com/doc/percona-server/5.6/performance/innodb_numa_support.html#numa_interleave>`_  ``mysql_safe.sh`` option have been replaced by `innodb_numa_interleave <http://dev.mysql.com/doc/refman/5.7/en/innodb-parameters.html#sysvar_innodb_numa_interleave>`_ server option. Note that :variable:`flush_caches` option still remains.
+ * `Ability to change database for mysqlbinlog <https://www.percona.com/doc/percona-server/5.6/flexibility/mysqlbinlog_change_db.html>`_ implementation has been replaced from |MariaDB| one with |MySQL| `rewrite-db <http://dev.mysql.com/doc/refman/5.7/en/mysqlbinlog.html#option_mysqlbinlog_rewrite-db>`_ one. The feature is mostly identical with two differences: 1) multiple rewrite rules must be given as separate options, and the ability to list them in a single rule, separated by commas, is lost. That is, any ``--rewrite-db='a->b,c->d'`` occurrences must be replaced with ``--rewrite-db='a->b' --rewrite-db='c->d'``. 2) Whitespace around database names is not ignored.
+ * `INFORMATION_SCHEMA.PROCESSLIST.TID column <https://www.percona.com/doc/percona-server/5.6/diagnostics/process_list.html>`_ has been has been replaced by `PERFORMANCE_SCHEMA.THREADS.THREAD_OS_ID column <http://dev.mysql.com/doc/refman/5.7/en/threads-table.html>`_. If running under thread pool, ``THREAD_OS_ID`` column will always be ``NULL``, where as in the 5.6 implementation ``TID`` column showed either ``NULL`` or the assigned worker thread id at the moment.
+ * `innodb_foreground_preflush server <https://www.percona.com/doc/percona-server/5.6/performance/xtradb_performance_improvements_for_io-bound_highly-concurrent_workloads.html#innodb_foreground_preflush>`_ variable has been removed as the upstream implemented a similar feature without a controlling option.
+ * `Log All Client Commands (syslog) <http://www.percona.com/doc/percona-server/5.6/diagnostics/mysql_syslog.html>`_ feature has been replaced by Oracle `mysql Logging <http://dev.mysql.com/doc/refman/5.7/en/mysql-logging.html>`_ implementation.
+ * Support for `Multiple user level locks per connection <https://www.percona.com/doc/percona-server/5.6/scalability/multiple_user_level_locks.html>`_ has been replaced by Oracle implementation, which is based on the same contributed patch by *Kostja Osipov*.
+ * `super-read-only option <https://www.percona.com/doc/percona-server/5.6/management/super_read_only.html>`_ has been replaced by Oracle `super_read_only <http://dev.mysql.com/doc/refman/5.7/en/server-system-variables.html#sysvar_super_read_only>`_ variable implementation.
+ * Mutex names in ``SHOW ENGINE INNODB MUTEX`` have been replaced by Oracle mutex name implementation.
+
+Features ported from |Percona Server| 5.6 to |Percona Server| 5.7
```

Comment:
I'd make this section very last one in this page (so that other removed/changed things come before)